python/tvm/testing.py

```diff
@@ -701,11 +717,79 @@ def _target_to_requirement(target):
     return []
 
 
+def _pytest_target_params(targets, excluded_targets=None, xfail_targets=None):
+    # Include unrunnable targets here.  They get skipped by the
+    # pytest.mark.skipif in _target_to_requirement(), showing up as
+    # skipped tests instead of being hidden entirely.
+    if targets is None:
+        if excluded_targets is None:
+            excluded_targets = set()
+
+        if xfail_targets is None:
+            xfail_targets = set()
+
+        target_marks = []
+        for t in _get_targets():
```

Comment:
Doesn't `_get_targets` filter out all non-unable targets? So we are not including unrunable targets here?

Reply:
The updated implementation of `_get_targets` returns all targets without filtering, but marks them as running or un-runnable.  This allows `enabled_targets()` to maintain its current behavior of filtering out un-runnable targets, while the `_pytest_target_params` can return all targets, but marked with `pytest.skipif` to indicate which ones cannot run on the current platform.